signinapp/model.py

```diff
@@ -381,6 +413,10 @@ def from_name(cls, name) -> Subteam:
         ' Get a subteam by name '
         return cls.query.filter_by(name=name).one_or_none()
 
+    @classmethod
+    def get_subteam(cls) -> List[str]:
```

Comment:
This isn't getting one subteam, this is getting all subteams. `get_subteams` instead (plural)

But is it even used?

Reply:
Updating this to match what you're doing in `admin.users.new_user` and `admin.users.edit_user`, and replacing those uses with calls to this function to ensure we're consistent.